### PR TITLE
added vt1 option, modified date string format

### PR DESCRIPTION
**Pull request by jsturdy**
* added VT1 setting option in ```trimChamber```
* updated zero VFATs to use new function in ```cmsgemos```
* updated the time/date format to be everywhere ```strftime("%Y.%m.%d.%H.%M")```

### CODE REVIEW

File: confChamber.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 from ROOT import TFile,TTree
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Comment (cbravo135) [inline confChamber.py]: This doesn't seem to be like the others. This is closer to what I would pick.

**Review by cbravo135 — COMMENTED** [review]
I would prefer a date format of %Y.%m.%d.%H.%M everywhere

File: confChamber.py (hunk 1 above)
Comment (jsturdy) [reply confChamber.py]: What's the difference?

**Comment by jsturdy** [comment]
Is that not what I changed everything to?  Ah, I see, hyphen vs. dash, well, I kept the old format and just removed the trailing seconds, and ordered as you had mentioned yesterday

**Comment by cbravo135** [comment]
"%d.%m.%Y-%H.%M" -> "%Y.%m.%d.%H.%M" the bigger difference is switching %Y and %d

File: VFATSCurveTools.py [hunk 2]
```diff
@@ -670,7 +670,7 @@ def runScanRoutine(self,vfat,debug=False):
     (options, args) = parser.parse_args()
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline VFATSCurveTools.py]: Here it is %d first

File: confChamber.py (hunk 1 above)
Comment (cbravo135) [reply confChamber.py]: here it is %Y first

File: pythonScript.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
         pass
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline pythonScript.py]: Here it is %d first

File: VFATSCurveTools.py (hunk 2 above)
Comment (jsturdy) [reply VFATSCurveTools.py]: Ah, got it

File: pythonScript_904.py [hunk 4]
```diff
@@ -25,7 +25,7 @@
     from ipbus import *
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline pythonScript_904.py]: Here it is %d first

File: run_scans.py [hunk 5]
```diff
@@ -16,19 +16,25 @@ def launchTestsArgs(tool, slot, link, chamber,vt2=0,perchannel=False,trkdata=Fal
...
+      pass
     preCmd = ["confChamber.py","-s%d"%(slot),"-g%d"%(link)]
     pass
   elif tool == "trimChamber.py":
     scanType = "trim"
     dataType = None
     pass
 
-  startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+  startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Comment (cbravo135) [inline run_scans.py]: Here it is %Y first

File: setTRIMDAC.py [hunk 6]
```diff
@@ -30,7 +30,7 @@
     from ipbus import *
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline setTRIMDAC.py]: Here it is %d first

**Comment by jsturdy** [comment]
Yeah, I didn't bother changing the scripts that aren't being used

File: setTRIMDAC_and_Mask.py [hunk 7]
```diff
@@ -28,7 +28,7 @@
     pass
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline setTRIMDAC_and_Mask.py]: Here it is %d first

File: setTRIMDAC_and_Scurve.py [hunk 8]
```diff
@@ -30,7 +30,7 @@
     from ipbus import *
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline setTRIMDAC_and_Scurve.py]: Here it is %d first

File: setTRIMDAC_and_Threshold.py [hunk 9]
```diff
@@ -32,7 +32,7 @@
     from ipbus import *
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline setTRIMDAC_and_Threshold.py]: Here it is %d first

File: trimChamber.py [hunk 10]
```diff
@@ -43,7 +45,7 @@ def runCommand(cmd):
 
 from macros.fitScanData import fitScanData
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Comment (cbravo135) [inline trimChamber.py]: Here it is %Y first

File: ultraScurve.py [hunk 11]
```diff
@@ -48,7 +48,7 @@
 link[0] = options.gtx
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Comment (cbravo135) [inline ultraScurve.py]: Here it is %Y first

File: ultraThreshold.py [hunk 12]
```diff
@@ -58,7 +58,7 @@
 link[0] = options.gtx
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Comment (cbravo135) [inline ultraThreshold.py]: Here it is %Y first

File: ultra_QC2.py [hunk 13]
```diff
@@ -23,7 +23,7 @@
 exit(1)
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Comment (cbravo135) [inline ultra_QC2.py]: Here it is %d first

**Review by cbravo135 — COMMENTED** [review]
Made explicit in line comments

File: takeScurves.py [hunk 14]
```diff
@@ -11,7 +11,7 @@
 (options, args) = parser.parse_args()
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d.%H.%M.%S.%f")
```
Comment (cbravo135) [inline takeScurves.py:14]: This one still has %S and %f

**Review by cbravo135 — APPROVED** [review]

**Pull request merged**